src/site/content/en/blog/how-we-build-webdev-and-use-web-components/index.md

```diff
@@ -0,0 +1,179 @@
+These include redirects for invalid domains, as well as code [to parse a user's preferred language](https://github.com/GoogleChrome/web.dev/blob/master/locale-handler.js) for an upcoming internationalization feature.
+
+{% Aside %}
+Despite being a Google product, web.dev doesn't use any internal Google tools or processes.
+You can check out [our repo](https://github.com/GoogleChrome/web.dev).
+We do this because the team that builds and maintains web.dev is responsible for advocating 
+within Google on behalf of the broader web developer ecosystem so it makes more sense for us
+to have the same tools as external web developers.
+{% endAside %}
+
+## Static generation
+
+Each page on web.dev is written in Markdown.
+All pages include [front matter](https://www.11ty.dev/docs/data-frontmatter/), which describes metadata about each post.
+This metadata is ingested into the layout of each page, creating headings, tags, and so on.
+Here's an example:
+
+```md
+---
+layout: post
+title: What is network reliability and how do you measure it?
+authors:
+  - jeffposnick
+date: 2018-11-05
+description: |
+  The modern web is enjoyed by a wide swath of people…
+---
+
+The modern web is enjoyed by a wide swath of [people](https://www.youtube.com/watch?v=dQw4w9WgXcQ), using a range of different devices and types of network connections.
+
+Your creations can reach users all across the world...
+```
+
+This front matter lets us define arbitrary properties like author(s), publish date, and tags.
+Eleventy conveniently exposes the front matter as [data](https://www.11ty.dev/docs/data/) in nearly every plugin, template, or other context where we'd like to do something intelligent.
+The data object also contains what Eleventy describes as the [data cascade](https://www.11ty.dev/docs/data-cascade/)—a variety of data pulled from each individual page, from the layout the page uses, and from data found in the hierarchical folder structure.
+
+Each unique layout describes a different type of content and can [inherit](https://www.11ty.dev/docs/layout-chaining/) from _other_ layouts.
+On web.dev, we use this feature to correctly frame different types of content (like posts and codelabs) while still sharing one top-level HTML layout.
+
+### Collections
+
+Eleventy provides a programmatic way to build arbitrary collections of content.
+This has let us build pagination support and generate virtual pages (pages that don't have a matching Markdown file on disk) for post authors.
+For example, we construct our authors pages using a template containing [an expression for its permalink](https://github.com/GoogleChrome/web.dev/blob/master/src/site/content/en/authors/index.njk#L4) (so the template is re-rendered for every author) and a backing [collection](https://github.com/GoogleChrome/web.dev/blob/master/src/site/_collections/paginated-posts-by-author.js#L23).
+
+This results in, for example, a simple page containing [all of Rob's posts](https://web.dev/authors/robdodson/)!
```

Comment:
are you cool switching this to Addy's page? I think it looks nicer with all of the hero images.